eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Utilities/TypeNameUtilities.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.TypeSpec.Generator;
+using System.Linq;
+
+namespace Azure.Generator.Management.Utilities
+{
+    internal static class TypeNameUtilities
+    {
+        private const string AzurePackageNamespacePrefix = "Azure.";
+        private const string AzureResourceManagerPackageNamespacePrefix = "Azure.ResourceManager.";
+
+        public static string GetResourceProviderName()
```

Comment:
we have a `ResourceProviderName` in the type factory.
Should we use that instead?